pyveda/tests/unit/test_datapoint.py

```diff
@@ -25,28 +25,28 @@ def setUp(self):
     def test_datapoint(self):
         dp = DataPoint(self.json)
         self.assertTrue(isinstance(dp, DataPoint))
-        assert dp.id == 'ae91f7df-ae37-4d31-9506-d9176f50403c' 
-        assert dp.mltype == 'classification' 
-        assert dp.dtype == 'uint8' # based on init logic and sample json
-        assert dp.label == {"building": 0}
-        assert dp.bounds == [
+        assertEqual(dp.id, 'ae91f7df-ae37-4d31-9506-d9176f50403c')
+        assertEqual(dp.mltype, 'classification')
+        assertEqual(dp.dtype, 'uint8') # based on init logic and sample json
+        assertEqual(dp.label, {"building": 0})
+        assertEqual(dp.bounds. [
```

Comment:
Needs a comma here

Reply:
this should be `self.assert....`!!!

Reply:
locally they run, and have `self.assert` I think that's not the most recent commit

Reply:
yeah, I just merged your branch and that looks fixed now

Reply:
merged locally, that is